internal/services/connection_check.go

```diff
@@ -160,12 +168,13 @@ func (cs *connectionService) connectionCheck() {
 			"connected": strconv.FormatBool(connected),
 		}
 
+		connection.With(labels).Inc()
```

Comment:
because of this https://github.com/strimzi/strimzi-canary/pull/216, a contributor had to initialize the error metric to 0. I think that we should do the same when it happens for the new connection metric, otherwise we'll have the same problem.

Reply:
Done, I've made it redundantly add `0` to the connection metrics (new and legacy) as soon as we know the broker id. I thought about keeping some state about which brokers have had metrics initialized and only doing it tactically but that felt like overkill.